cmd/snap-update-ns/utils.go

```diff
@@ -362,6 +362,93 @@ func planWritableMimic(dir string) ([]*Change, error) {
 	return changes, nil
 }
 
+// execWritableMimic executes the plan for a writable mimic.
+// The result is a transformed mount namespace and a set of fake mount changes
```

Comment:
Got slightly confused with the wording, or maybe I'm just missing a wider context here. The idea is that the function returns a list of changes that need to be 'undone' when cleaning up. Did I get this right?

Reply:
A list of changes that can be handled by the undo algorithm.

Reply:
Got it. Thanks for explaining.